.circleci/config.yml

```diff
@@ -0,0 +1,246 @@
+version: 2
+
+# ################## WORKFLOWS ##################
+
+workflows:
+  version: 2
+  build:
+    jobs:
+      - build
+      - lms_unit_tests:
+          requires:
+            -  build
+      - cms_unit_tests:
+            requires:
+            -  build
+      - lib_unit_tests:
+            requires:
+            -  build
+      - javascript_tests:
+            requires:
+            -  build
+
+# ################## JOBS ##################
```

Comment:
@clemente nit: I don't think this is needed, it's clear that this is the jobs section, thoughts?

Reply:
Ops! That's the bad part of having two people reviewing! I'm 2nd one, so please follow @rocioar's comments first.

I think some visual separation helps to browse the file, but it could be a simple `# -------------------------` everywhere if we don't want to repeat the section name.
Would that work?

Those comments don't need to explain, just to aid visually (and be highlighted in another color).

Reply:
I think moving the workflow declaration from the bottom to the top of the file has made the structure of the rest of the file clearer than it was before. So will go with removing these lines for now. :)

Reply:
agree @symbolist